testing.md

```diff
@@ -31,6 +32,61 @@ You are free to define other testing environment configuration values as necessa
 
 In addition, you may create a `.env.testing` file in the root of your project. This file will be used instead of the `.env` file when running Pest and PHPUnit tests or executing Artisan commands with the `--env=testing` option.
 
+<a name="caching-config"></a>
+## Caching Config
+
+When running tests, Laravel boots the application for each individual test method.  Without a cached configuration file, each config file for your application must be loaded at the start of a test. To build the configuration once and re-use it for all tests in a single run, Laravel offers the `Illuminate\Foundation\Testing\WithCachedConfig` trait. 
+
+```php tab=Pest
+<?php
+
+use Illuminate\Foundation\Testing\WithCachedConfig;
+
+pest()->use(WithCachedConfig::class);
+
+test('modifies config', function () {
+    config(['services.postmark.key' => 'xyz']);
+    expect(config('services.postmark.key'))->toBe('xyz');
+});
+
+test('uses default config', function () {
+    expect(config('services.postmark.key'))->not->toBe('xyz');
+});
+```
+
+```php tab=PHPUnit
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\WithCachedConfig;
+use Tests\TestCase;
+
+class ConfigTest extends TestCase
+{
+    use WithCachedConfig;
+
+    /**
+     * A test that modifies the config.
+     */
+    public function test_modifies_config(): void
+    {
+        $this->assertEquals('xyz', config('services.postmark.key'));
+    }
```

Comment:
FYI this PHPUnit example seems to be missing a call to `config(['services.postmark.key' => 'xyz']);` to be equivalent to the Pest test example.